src/poliastro/tests/tests_twobody/test_perturbations.py

```diff
@@ -101,3 +107,63 @@ def accel(t0, state, k):
     final = initial.propagate(initial.period, method=cowell, ad=accel)
     assert_quantity_allclose(final.r, initial.r)
     assert_quantity_allclose(final.v, initial.v)
+
+
+def test_moon_at_right_position():
+    epoch = Time(2456498.8333, format='jd', scale='tdb')
+    moon = Orbit.from_body_ephem(Moon, epoch)
+    moon = change_frame(moon, ICRS, GCRS)
+    r_expected = np.array([340958.0, -137043.0, -27521.3]) * u.km
+
+    assert_quantity_allclose(moon.r, r_expected, rtol=1e-2)
+
+
+def test_3rd_body():
+    # example 12.11 from Howard Curtis
+    epoch = Time(2454283.0, format='jd', scale='tdb')
+
+
+    # propagate Moon in the interval (0, tof) and write it into the OdeSolution object
+    tof = (60 * u.day).to(u.s).value
+
+    moon = Orbit.from_body_ephem(Moon, epoch)
+    moon = change_frame(moon, ICRS, GCRS)
+
+    k = moon.attractor.k.to(u.km ** 3 / u.s ** 2).value
+    x, y, z = moon.r.to(u.km).value
+    vx, vy, vz = moon.v.to(u.km / u.s).value
+
+    u0 = np.array([x, y, z, vx, vy, vz])
+
+    # Set the non Keplerian acceleration
+    ad = lambda t0, u_, k_: (0, 0, 0)
+    f_with_ad = functools.partial(func_twobody, k=k, ad=ad, ad_kwargs=dict())
+
+    moon_dense = solve_ivp(f_with_ad, (0, tof), u0,
+                           rtol=1e-8, atol=1e-12, method=DOP835,
+                           dense_output=True)
+
+    ecc = 0.741 * u.one
+    a = 26553.4 * u.km
+    raan = 0.0 * u.deg
+    inc = 63.4 * u.deg
+    argp = 270 * u.deg
+    nu = 0.0 * u.rad
+
+    initial = Orbit.from_classical(Earth, a, ecc, inc, raan, argp, nu, epoch=epoch)
+
+    r, v = cowell(initial, tof, rtol=1e-8, ad=third_body, 
+                  k_third=Moon.k.to(u.km**3 / u.s**2).value, third_body=moon_dense)
+    _, _, _, raan, _, _ = rv2coe(Earth.k.to(u.km**3 / u.s**2).value, r, v)
+
+    assert_quantity_allclose((raan * u.rad).to(u.deg) - 360 * u.deg, -0.06 * u.deg, rtol=1e-2)
+
+
+def test_moon_at_right_position():
+    # based on Cowell, example 12.10
+    epoch = Time(2456498.8333, format='jd', scale='tdb')
+    moon = Orbit.from_body_ephem(Moon, epoch)
+    moon = change_frame(moon, ICRS, GCRS)
+    r_expected = np.array([340958.0, 137043.0, 27521.3]) * u.km
```

Comment:
These numbers are wrong, see HORIZONS:

![screenshot-2018-6-1 horizons web-interface](https://user-images.githubusercontent.com/316517/40828648-81a581fe-6581-11e8-961a-9d7f3bdf522d.png)

and Curtis:

![screenshot from 2018-06-01 09-52-51](https://user-images.githubusercontent.com/316517/40828685-9ce9bb38-6581-11e8-80e6-db674742cf04.png)